### PR TITLE
Add fleet vehicle cancelation

### PR DESCRIPTION
**Pull request by scudelletti**
Adds fleet vehicle cancelation endpoint.

### CODE REVIEW

File: hm-service-account-api-rest-v1.yml [hunk 1]
```diff
@@ -294,6 +294,43 @@ paths:
...
+    post:
+      security:
+        - ServiceAccountToken: []
+      tags:
+        - Fleet
+      responses:
+        '200':
+          schema:
+            $ref: '#/definitions/FleetVin'
+          description: Vehicle will be canceled soon
```
Comment (kevinvaldek) [inline hm-service-account-api-rest-v1.yml:307]: Proposal: "Vehicle activation is being canceled"

File: hm-service-account-api-rest-v1.yml [hunk 2]
```diff
@@ -294,6 +294,43 @@ paths:
...
+        '422':
+          schema:
+            $ref: '#/definitions/FleetErrors'
+          description: Invalid Input. It occurs when the payload is missing a params or the the application is not configured.
```
Comment (kevinvaldek) [inline hm-service-account-api-rest-v1.yml:311]: Small typo with double "the" in the text. Can use:

"Invalid input. This occurs when the payload is missing a parameter or the application has not been configured."

File: hm-service-account-api-rest-v1.yml [hunk 3]
```diff
@@ -294,6 +294,43 @@ paths:
...
+        '500':
+          description: Server Errors
+      parameters:
+        - type: string
+          required: true
+          name: vin
+          in: path
+          description: Vehicle identification number
+      operationId: ApiWeb.FleetCancelActivationController.create
+      description: Enqueue fleet vehicle cancelation
```
Comment (kevinvaldek) [inline hm-service-account-api-rest-v1.yml:333]: Description propsal: "Cancel the ongoing activation process of a vehicle that has the "pending" status. Note that this endpoint does not revoke vehicles that are in the "approved" state, for which the OAuth2 API has to be used."

Other than that looks good! Please ping me once it's live and I'll make sure to update our docs!

File: hm-service-account-api-rest-v1.yml (hunk 1 above)
Comment (scudelletti) [reply hm-service-account-api-rest-v1.yml]: Done

File: hm-service-account-api-rest-v1.yml (hunk 2 above)
Comment (scudelletti) [reply hm-service-account-api-rest-v1.yml]: Done

File: hm-service-account-api-rest-v1.yml (hunk 3 above)
Comment (scudelletti) [reply hm-service-account-api-rest-v1.yml]: Done

**Review by kevinvaldek — APPROVED** [review]

**Pull request merged**